src/sssom/parsers.py

```diff
@@ -212,21 +212,28 @@ def parse_sssom_table(
                     logging.info(f"Externally provided metadata {k}:{v} is added to metadata set.")
                     sssom_metadata[k] = v
         meta = sssom_metadata
-
-        if "curie_map" in sssom_metadata:
+        if CURIE_MAP in sssom_metadata:
```

Comment:
I have rewritten all of this code using `curies` already and am waiting for #426 (ready) then #401 (needs some more work) to get merged. I think it's better that you reduce effort on re-writing it again, since the `curies` solution is much more clean and elegant.

Reply:
I split the code for updating this into #431